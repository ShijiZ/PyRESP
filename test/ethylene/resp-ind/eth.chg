All values are reported in atomic units
%FLAG TITLE
 molecule 1: ethylene

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1   0.0000000E+00   0.0000000E+00   1.2441670E+00
   2   0.0000000E+00   0.0000000E+00  -1.2441670E+00
   3   0.0000000E+00   1.7277940E+00   2.3159880E+00
   4   0.0000000E+00  -1.7277940E+00  -2.3159880E+00
   5   0.0000000E+00   1.7277940E+00  -2.3159880E+00
   6   0.0000000E+00  -1.7277940E+00   2.3159880E+00

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          6          0    -6.2730312E-01
   2          6          1    -6.2730312E-01
   3          1          0     3.1371545E-01
   4          1          3     3.1371545E-01
   5          1          3     3.1371545E-01
   6          1          3     3.1371545E-01

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   0.0000000E+00   0.0000000E+00  -3.9999144E-01
   2   0.0000000E+00  -1.7347235E-18   3.9999144E-01
   3   0.0000000E+00  -9.3585117E-02  -5.2172756E-02
   4   0.0000000E+00   9.3585117E-02   5.2172756E-02
   5   0.0000000E+00  -9.3585117E-02   5.2172756E-02
   6   0.0000000E+00   9.3585117E-02  -5.2172756E-02

