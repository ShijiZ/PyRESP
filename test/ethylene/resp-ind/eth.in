test for ethylene
 &cntrl
 nmol = 1,
 iqopt = 1,
 ihfree = 1,
 qwt = 0.0005,
 ioutopt = 1,
 ipol = 5,
 igdm = 1,
 exc12 = 0,
 exc13 = 0,
 ipermdip = 0,
 &end 
    1.0
ethylene
   -99   6
    6    0
    6    1
    1    0
    1    3
    1    3
    1    3




