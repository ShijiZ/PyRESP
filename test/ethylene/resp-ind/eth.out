
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 test for ethylene
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name: ethylene
 Total charge (ich):-99
 Number of centers:  6
    1    6    0
    2    6    1
    3    1    0
    4    1    3
    5    1    3
    6    1    3

 -----------------------------
 Atom Ivary Information
 -----------------------------
    6    0
    6    1
    1    0
    1    3
    1    3
    1    3

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =    6
 Weight factor on initial charge restraints=  0.000500
 There are  0 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =     6
 total number of esp points =   452

 center       X               Y               Z
    1   0.0000000E+00   0.0000000E+00   1.2441670E+00
    2   0.0000000E+00   0.0000000E+00  -1.2441670E+00
    3   0.0000000E+00   1.7277940E+00   2.3159880E+00
    4   0.0000000E+00  -1.7277940E+00  -2.3159880E+00
    5   0.0000000E+00   1.7277940E+00  -2.3159880E+00
    6   0.0000000E+00  -1.7277940E+00   2.3159880E+00


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 values will be set to 0

 Number of unique UNfrozen charge centers =     2

 Non-linear optimization requested.
  qchnge =    1.8348055153E-01
  qchnge =    2.2998897106E-03
  qchnge =    7.8732522105E-05
  qchnge =    2.7280936574E-06
  qchnge =    9.4568278648E-08

 Convergence in    4 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: ethylene

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         6        0.000000     -0.627303      0       0.000787
    2         6        0.000000     -0.627303      1       0.000787
    3         1        0.000000      0.313715      0       0.000000
    4         1        0.000000      0.313715      3       0.000000
    5         1        0.000000      0.313715      3       0.000000
    6         1        0.000000      0.313715      3       0.000000

 Sum over the calculated charges:      0.000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          2.0365103E-02
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   2.1210371E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               2.1662317E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         3.2272367E-01

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        0.00000    0.00000    0.00000   -0.00000

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   -1.92073
        Y    0.00000    1.37804
        Z    0.00000   -0.00000    0.54269

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X    1.37804
        Y    0.00000    0.54269
        Z    0.00000    0.00000   -1.92073